perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -257,8 +261,19 @@ def RemoteCommand(self, command,
                     should_log=False, retries=SSH_RETRIES,
                     ignore_failure=False, login_shell=False,
                     suppress_warning=False):
+    return self.RemoteHostCommand(command, should_log, retries,
+                                  ignore_failure, login_shell,
+                                  suppress_warning)
+
+  def RemoteHostCommand(self, command,
```

Comment:
The issue here is that RemoteHostCommand, RemoteHostCopy, and MoveHostFile are used outside of the containerized vm class. Namely in methods like FormatDisk, WaitForBootCompletion, and other stuff within BaseLinuxMixin